pattern/variable/SystemReference.java

```diff
@@ -18,6 +18,7 @@
 
 package grakn.core.pattern.variable;
 
+import graql.lang.common.GraqlToken;
```

Comment:
You add GraqlToken, but not actually use it, @flyingsilverfin ?

Reply:
oof